TelephoneBook/main.c

```diff
@@ -0,0 +1,120 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include <string.h>
+#include <locale.h>
+#include "test.h"
+#include "telephonebook.h"
+#define maxLength 150 
```

Comment:
```suggestion
#define MAX_LENGTH 150 
```
Объявления через `#define` по стайлгайду именуются `КАПСОМ_С_ПОДЧЁРКИВАНИЕМ`